Makefile

```diff
@@ -210,20 +210,19 @@ generate-go-conversions: ## Run all generate-go-conversions-* targets
 generate-go-conversions-rke2-bootstrap: $(CONVERSION_GEN) ## Generate conversions go code for the rke2 bootstrap
 	$(MAKE) clean-generated-conversions SRC_DIRS="./bootstrap/api/v1alpha1"
 	$(CONVERSION_GEN) \
-		--input-dirs=./bootstrap/api/v1alpha1 \
-		--build-tag=ignore_autogenerated_rke2_bootstrap \
-		--output-file-base=zz_generated.conversion $(ROOT_DIR) \
-		--go-header-file=./hack/boilerplate.go.txt
+		--output-file=zz_generated.conversion.go $(ROOT_DIR)/$(CAPBPR_DIR) \
+		--go-header-file=./hack/boilerplate.go.txt \
+		./bootstrap/api/v1alpha1
 
 .PHONY: generate-go-conversions-rke2-control-plane
 generate-go-conversions-rke2-control-plane: $(CONVERSION_GEN) ## Generate conversions go code for the rke2 control plane
 	$(MAKE) clean-generated-conversions SRC_DIRS="./controlplane/api/v1alpha1"
 	$(CONVERSION_GEN) \
-		--input-dirs=./controlplane/api/v1alpha1 \
-		--extra-peer-dirs=github.com/rancher/cluster-api-provider-rke2/bootstrap/api/v1alpha1 \
+		--extra-dirs=github.com/rancher/cluster-api-provider-rke2/bootstrap/api/v1alpha1 \
 		--build-tag=ignore_autogenerated_rk2_control_plane \
-		--output-file-base=zz_generated.conversion $(ROOT_DIR) \
-		--go-header-file=./hack/boilerplate.go.txt
+		--output-file=zz_generated.conversion.go $(ROOT_DIR)/$(CAPRKE2_DIR) \
```

Comment:
I think the `--build-tag` here need to be removed like the bootstrap?

Reply:
updated 👍